src/components/Layouts/Connected/Connected.layout.jsx

```diff
@@ -56,7 +56,12 @@ const ConnectedLayout = () => {
           </Box>
         </Drawer>
       </Box>
-      <Main open={open}>
+      <Main
+        open={open}
+        sx={{
+          paddingLeft: '40px',
+        }}
```

Comment:
Please remove this part; you can set up the wanted spacing in `Connected.layout.js` file in the styled component Main line `19` setup it to `padding: theme.spacing(4),` for example